clickhouse_sqlalchemy/drivers/base.py

```diff
@@ -307,6 +309,52 @@ def visit_insert(self, insert_stmt, asfrom=False, **kw):
             rv = rv[:pos + 6]
         return rv
 
+    def visit_update(self, update_stmt, asfrom=False, **kw):
+        extra_froms = update_stmt._extra_froms
+        is_multitable = bool(extra_froms)
```

Comment:
It seems that multitable update is not supported in ClickHouse now. We can simplify `visit_update` by removing it.